hail/src/test/scala/is/hail/io/fs/FSSuite.scala

```diff
@@ -42,45 +44,56 @@ trait FSSuite extends TestNGSuite {
 
   def pathsRelResourcesRoot(statuses: Array[FileListEntry]): Set[String] = pathsRelRoot(fsResourcesRoot, statuses)
 
-  @Test def testExists(): Unit = {
-    assert(fs.exists(r("/a")))
-
-    assert(fs.exists(r("/zzz")))
-    assert(!fs.exists(r("/z"))) // prefix
-
+  @Test def testExistsOnDirectory(): Unit = {
     assert(fs.exists(r("/dir")))
     assert(fs.exists(r("/dir/")))
 
     assert(!fs.exists(r("/does_not_exist")))
     assert(!fs.exists(r("/does_not_exist_dir/")))
   }
 
+  @Test def testExistsOnFile(): Unit = {
+    assert(fs.exists(r("/a")))
+
+    assert(fs.exists(r("/zzz")))
+    assert(!fs.exists(r("/z"))) // prefix
+  }
+
+  @Test def testFileStatusOnFile(): Unit = {
+    // file
+    val f = r("/a")
+    val s = fs.fileStatus(f)
+    assert(s.getPath == f)
+    assert(s.getLen == 12)
+  }
+
   @Test def testFileListEntryOnFile(): Unit = {
     // file
     val f = r("/a")
     val s = fs.fileListEntry(f)
     assert(s.getPath == f)
-    assert(s.isFile)
-    assert(!s.isDirectory)
```

Comment:
Why did you get rid of these tests here? They seem valuable to me.

Reply:
mistake, restored.